.changeset/kind-bees-suffer.md

```diff
@@ -0,0 +1,32 @@
+---
+'@backstage/plugin-catalog-backend': minor
+'@backstage/plugin-permission-common': minor
+'@backstage/plugin-permission-node': minor
+'@backstage/plugin-playlist-backend': minor
```

Comment:
Please break this up into additional changesets. The contents here doesn't really make sense for the catalog and playlist backends.